### ISSUE TITLE
Add one cycle LR to hypervalidation

### ISSUE THREAD

**Issue by sternakt** [airtai/tpk#9]
- [x] Add OneCycleLR https://pytorch.org/docs/stable/generated/torch.optim.lr_scheduler.OneCycleLR.html
- [x] Calculate learning rate using LR finder: https://pytorch-lightning.readthedocs.io/en/1.4.5/advanced/lr_finder.html
- [x] Add "epoch_num" hyperparameter to trial and remove early stopping
- [x] Refactor LR to separate CLI
- [x] Run LR finder in Trial and add it to model train runs
